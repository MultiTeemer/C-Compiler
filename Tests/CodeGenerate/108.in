int main()
{	
	printf("%f %f", 1.3, 10.3453);
}