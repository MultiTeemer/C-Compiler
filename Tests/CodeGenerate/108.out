1.300000 10.345300